### PR TITLE
Added new Flux contrib/events package

### PR DESCRIPTION
**Pull request by sanderson**
- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

**Comment by sanderson** [comment]
@tomhollingworth, you're welcome to look at this documentation PR for your Events package.

**Comment by tomhollingworth** [comment]
Thanks @sanderson! I'll check it out

**Review by kelseiv — APPROVED** [review]

File: content/influxdb/v2.0/reference/flux/stdlib/contrib/events/duration.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+
+{{% note %}}
+#### Similar functions
+`events.duration()` is similar to `elapsed()` and `stateDuration()`, but differs
+in important ways:
+
+- `elapsed()` drops the first record. `events.duration()` does not.
+- `stateDuration()` calculates the total time spent in a state (determined by a
+  [predicate function](/influxdb/v2.0/reference/glossary/#predicate-function)).
+  `events.duration()` returns the duration between all records and their subsequent records.
```
[the remark this answers is not in the document]
Comment (sanderson) [reply content/influxdb/v2.0/reference/flux/stdlib/contrib/events/duration.md:45]: No, it's the 2nd part to explaining how `events.duration` is different from `stateDuration`.

File: content/influxdb/v2.0/reference/flux/stdlib/contrib/events/duration.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+
+For examples, see [below](#compared-to-similar-functions).
+{{% /note %}}
+
+## Parameters
+
+### unit
+<span class="req">Required</span>
```
Comment (tomhollingworth) [inline content/influxdb/v2.0/reference/flux/stdlib/contrib/events/duration.md:53]: `unit` isn't required, Its default is "1ns"

Comment (sanderson) [reply content/influxdb/v2.0/reference/flux/stdlib/contrib/events/duration.md:53]: Thanks @tomhollingworth, fixed.

**Review by tomhollingworth — COMMENTED** [review]
Well written and easy to understand, thanks @sanderson! The only thing I picked up on regarding the content is that `unit` isn't a required field and defaults to _1ns_.

**Comment by sanderson** [comment]
@tomhollingworth Thanks for the source docs you provided! Made it a ton easier to document.

**Pull request merged**